source/patterns/@aws-solutions-constructs/aws-s3-step-function/test/integ.pre-existing-bucket.ts

```diff
@@ -15,16 +15,15 @@
 import { App, Stack, RemovalPolicy } from "@aws-cdk/core";
 import { S3ToStepFunction, S3ToStepFunctionProps } from "../lib";
 import * as stepfunctions from '@aws-cdk/aws-stepfunctions';
-import * as s3 from '@aws-cdk/aws-s3';
 import { CreateScrapBucket } from '@aws-solutions-constructs/core';
-import { generateIntegStackName } from '@aws-solutions-constructs/core';
+import { generateIntegStackName, addCfnNagS3BucketNotificationRulesToSuppress } from '@aws-solutions-constructs/core';
 
 const app = new App();
 const stack = new Stack(app, generateIntegStackName(__filename));
 
-const existingBucket = CreateScrapBucket(stack, {});
-const cfnBucket = existingBucket.node.defaultChild as s3.CfnBucket;
-cfnBucket.addPropertyOverride('NotificationConfiguration.EventBridgeConfiguration.EventBridgeEnabled', true);
+const existingBucket = CreateScrapBucket(stack, {
+  eventBridgeEnabled: true
```

Comment:
Is there now a requirement that if an existing bucket is provided that eventBridgeEnabled is set to true? We should document that in the README. Better yet, can we turn it on ourselves? (I'm guessing that if this accepts an IBucket then we are unable to do so...)